openapi/sms/v1-review/partner-service.yaml

```diff
@@ -0,0 +1,364 @@
+openapi: 3.0.3
+info:
+  title: SAP Emarsys SMS Partner Service
+  description: |
+    An SAP Emarsys SMS Partner Service allows partners to host SMS message sends that are triggered
+    by an SAP Emarsys SMS campaign launch. This OpenAPI specification describes the required Partner service endpoints.
+  version: 0.0.1
+  contact:
+    name: SAP Emarsys SMS Team
+    email: team-sms@emarsys.com
+x-sap-shortText: Configure, test client setup and sends SMS messages.
+servers:
+  - url: https://api.sms-partner.io/sms/partner/v1
+paths:
+  /clients/{clientId}/configuration:
+    summary: Manage a client configuration within a Partner service
+    parameters:
+      - $ref: './common.yaml#/components/parameters/clientIdPathParam'
+    put:
+      tags: [ Client Configuration ]
+      operationId: createOrUpdateClientConfiguration
+      summary: Create or update a single client's configuration
+      requestBody:
+        description: client's custom field values as a map.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateOrUpdateClientConfigurationRequest'
+      responses:
+        '200':
+          description: Client configuration was successfully recorded within the Partner service.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreateOrUpdateClientConfigurationResponse'
+        '400':
+            $ref: '#/components/responses/InvalidClientConfigurationFormatResponse'
+        '401':
+          $ref: './common.yaml#/components/responses/AuthenticationFailedResponse'
+        '403':
+          $ref: './common.yaml#/components/responses/AuthorizationFailedResponse'
+        '5XX':
+          $ref: './common.yaml#/components/responses/ServerErrorResponse'
+    get:
+      tags: [ Client Configuration ]
+      operationId: getClientConfiguration
+      summary: Get a client's configuration by their ID
+      responses:
+        '200':
+          description: Client exists within the Partner service and their configuration
+            is returned in the response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetClientConfigurationResponse'
+        '403':
+          description: Authentication failed.
+        '404':
+          description: Client does not have a configuration recorded within the Partner service.
+        '5XX':
+          description: Server error occurred
+          content:
+            application/json:
+              schema:
+                $ref: './common.yaml#/components/schemas/Error'
```

Comment:
This error response only has message field and is not consistent with any other error that this API has. Also not up to SAP standard.
Same for other endpoints so not gonna repeat everywhere..